src/Sylius/Bundle/AttributeBundle/Controller/AttributeController.php

```diff
@@ -0,0 +1,75 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Sylius\Bundle\AttributeBundle\Controller;
+
+use Sylius\Bundle\AttributeBundle\AttributeType\TextAttributeType;
+use Sylius\Bundle\ResourceBundle\Controller\ResourceController;
+use Sylius\Component\Attribute\Model\AttributeInterface;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\HttpFoundation\Response;
+
+/**
+ * @author Mateusz Zalewski <mateusz.zalewski@lakion.com>
+ */
+class AttributeController extends ResourceController
+{
+    /**
+     * @return Response
+     */
+    public function renderAttributeTypesAction()
+    {
+        $attributeTypes = $this->get('sylius.registry.attribute_type')->all();
+
+        return $this->render('SyliusWebBundle:Backend/ProductAttribute:attributeTypesModal.html.twig', array('attributeTypes' => $attributeTypes));
```

Comment:
Why do we have `SyliusWebBundle:Backend/ProductAttribute:attributeTypesModal.html.twig` hardcoded? Is `ProductAttribute` bounded with this controller?